src/core/parcel-validation.ts

```diff
@@ -0,0 +1,146 @@
+import { generateLazyValidator, JSONSchema, ValidateFunction } from '../validation'
+import { isOutOfBoundsParcel } from './parcel-exceptions'
+import { KeywordDefinition } from 'ajv'
+
+/**
+ * Represents a parcel with X and Y coordinates
+ * @public
+ */
+export type Parcel = { x: number; y: number }
+
+/**
+ * Standard limits for parcels in Decentraland
+ * @public
+ */
+export const PARCEL_LIMITS = {
+  minX: -150,
+  maxX: 150,
+  minY: -150,
+  maxY: 150
+}
+
+/**
+ * Namespace with utilities to validate parcels
+ * @public
+ */
+export namespace Parcel {
+  /**
+   * Custom validation to check if coordinates are within limits
+   */
+  export const _isInLimits: KeywordDefinition = {
+    keyword: '_isInLimits',
+    validate: function validate(schema: boolean, data: Parcel) {
+      if (!data || typeof data.x !== 'number' || typeof data.y !== 'number') {
+        return false
+      }
+
+      const { x, y } = data
+      // Check if it's within the standard limits
+      const inStandardLimits =
+        x >= PARCEL_LIMITS.minX && x <= PARCEL_LIMITS.maxX && y >= PARCEL_LIMITS.minY && y <= PARCEL_LIMITS.maxY
+
+      // If it's within the standard limits, it's valid
+      if (inStandardLimits) {
+        return true
+      }
+
+      // If it's not within the standard limits, check exceptions
+      const parcelString = `${x},${y}`
+      return isOutOfBoundsParcel(parcelString)
+    },
+    errors: false
+  }
+
+  export const schema = {
+    type: 'object',
+    required: ['x', 'y'],
+    properties: {
+      x: {
+        type: 'number'
+      },
+      y: {
+        type: 'number'
+      }
+    },
```

Comment:
The schema seems to be missing the `additionalProperties` property to prevent it from having other properties besides x and y:
```suggestion
       additionalProperties: false,
    },
```